update for multiple channels!

TheWildBear opened the pull request:
I updated the bot so it can watch multiple channels! it also tells which users are on currently!

TheWildBear committed:
update for overview of multiple channels

TheWildBear committed:
remove logging and tell who's on

jeyemwey commented:
Hi, thank you for your Pull Request. I will look into the code and test it on my servers.

I'm currently writing my summer exams, so please don't expect any reaction for the next weeks.

jeyemwey committed:
Code Style enhancements

* Remove unused utils-library
* Move objectLength to the other helper functions
* Apply Bracket rules as seen in the rest of the project
* Read botUsers from the `loggedInUsers.log` file instead of a hardcoded user
* Instead of `botusername`, use `process.env.MUMBLEUSER`.
* Add Catch-Block to sendMessage method to avoid this:
```
(node:7) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): [object Object]
(node:7) DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```
* Re-Added the rootCh object, so messages from telegram can get relayed into the mumble chat.

jeyemwey committed:
Read existing users from log instead of hardcoding chat ids.
